src/components/Footer/index.tsx

```diff
@@ -3,14 +3,18 @@ import { FC } from 'react'
 const TSBLogo = '/images/tsb-logo.svg'
 const CityLABLogo = '/images/citylab-logo.svg'
 const BerlinLogo = '/images/berlin-skzl-logo.svg'
-const OdisLogo = '/images/logo-odis-berlin-black.svg'
+const OdisLogo = '/images/logo-odis-berlin.svg'
+const SenWEBLogo = '/images/logo-berlin-senweb-de.svg'
+const IBBLogo = '/images/ibb-logo.svg'
+const SenFINLogo = 'images/logo-berlin-senfin.svg'
+// const SenFINLogo = '/images/logo-odis-berlin-black.svg'
```

Comment:
This is probably meant to be removed, right?

Reply:
Yes, sorry, I removed it Thanks for the review!